### ISSUE TITLE
Weird tooltip bug after 10.0.2

### ISSUE THREAD

**Issue by Nillx** [Backupiseasy/ThreatPlates#368]
Since wow update 10.0.2 i started seeing stuff like this. 
![202316616-208cf367-3f20-4fb0-afce-51cca3aa6c23](https://user-images.githubusercontent.com/34476875/202323743-d262788d-ccf7-4015-b6a4-4839a84bdda4.png)
![202311102-b626d5b6-d3bf-4a84-9e0e-5caddb74e1b1](https://user-images.githubusercontent.com/34476875/202323747-8fcc853e-bd19-4ac4-b3a9-1cd640a7b409.png)
![202183516-1eb7ebc6-d365-4334-b211-fff8470d0a20](https://user-images.githubusercontent.com/34476875/202323748-fffdeb2a-fbc3-4e3f-b765-75466a45c1d4.png)

At first i thought it was a Leatrix Plus issue but then i noticed that its happening with Rarity as well. This is happening only when Threat Plates is enabled.

 Was able to reproduce it in different locations. Noticed that its often happening when there is an active name plate on the screen. Go in a place with a lot of enemy NPCs and start turning around your camera while targeting yourself, then check the tooltip if you have an addon that shows the target (Like Leatrix Plus).

**Comment by Backupiseasy** [comment]
Not sure why that should be happening. Threat Plates does not really modify tooltips except creating aura tooltips (which the above is not). Hm ...

**Comment by Backupiseasy** [comment]
I will look into https://wowpedia.fandom.com/wiki/Patch_10.0.2/API_changes#Tooltip_Changes - maybe, my best guess currently, it is related to these changes.

**Comment by Nillx** [comment]
I really wish i was able to help

**Comment by Backupiseasy** [comment]
For, it's always okay if I at least can reproduce the bug and then have an idea why a bug is occuring. So, this is fine here. There are other issues where I am completely lost why they happen. That is what is really frustrating ...

**Comment by Nillx** [comment]
For now im not seeing that issue with this version. Will post again if i see it happen again.

**Comment by benhamlin** [comment]
I'm seeing this issue and I don't use Threat Plates, so it may be a lib or a core game issue.

**Comment by Nillx** [comment]
Interesting, it was happening only when TP is enabled and there are nameplates on the screen but its a bit difficult to reproduce so maybe i missed it.

**Comment by Backupiseasy** [comment]
Threat Plates has embedded some libraries, e.g., Ace3. If the error is on one of these libs, then they would occur even when only TP is enabled. I think I will check these addons as well. 

I know that there is an updated (alpha) version of Ace3 which changed some tooltip code, but I still cannot believe that this wuold be the cause of this bug.

**Comment by mkarajohn** [comment]
If it helps, I am getting this in the bug trace

```
756x SharedXML/Tooltip/TooltipUtil.lua:8: attempt to index local 'tbl' (a nil value)
[string "@SharedXML/Tooltip/TooltipUtil.lua"]:8: in function <SharedXML/Tooltip/TooltipUtil.lua:7>
[string "@TidyPlates_ThreatPlates/Functions/healthtext.lua"]:90: in function <...Ons/TidyPlates_ThreatPlates/Functions/healthtext.lua:77>
[string "@TidyPlates_ThreatPlates/Functions/healthtext.lua"]:227: in function `func'
[string "@TidyPlates_ThreatPlates/Functions/healthtext.lua"]:329: in function `SetCustomText'
[string "@TidyPlates_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua"]:851: in function <...s_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua:845>
[string "@TidyPlates_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua"]:406: in function <...s_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua:376>
[string "@TidyPlates_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua"]:467: in function <...s_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua:445>
[string "@TidyPlates_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua"]:1127: in function `?'
[string "@TidyPlates_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua"]:1081: in function <...s_ThreatPlates/TidyPlatesInternal/TidyPlatesCore.lua:1080>

Locals:
tbl = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'tbl' (a nil value)"
```

**Comment by Backupiseasy** [comment]
Does this happen on Retail or on Classic?

**Comment by mkarajohn** [comment]
Sorry for late reply, it happened on retail.

**Comment by Backupiseasy** [comment]
Fixed with release 11.1.2 - I never saw this again after that.
